overview.qmd

```diff
@@ -21,4 +21,34 @@ We will not use the course page on Blackboard. Instead, all course materials can
+6  Re-usable R code (with R packages and Shiny)
+
+Each topic includes a lecture with tutorial elements (or a seminar) and one or more exercises. Some topics have an associated deliverable (graded assignment). 
+
+For example, for the topic of reproducible research, the lecture slides discuss the definition of reproducibility, what reproducibility can look like in the context of statistical research, and the requirements for reproducible research compendiums. The exercise is a tutorial on developing research compendiums with R and Quarto. And the deliverable is a complete and reproducible research compendium for a statistical research project.
+
+### Lectures
+Depending on the topic, the lecture contains some interactive elements, or will have a showcase/walkthrough of a development workflow (e.g. R package development). The lectures contribute to learning objective a: ‘Students develop fundamental knowledge and understanding in the state of the art in statistical markup languages and reproducible programming and development’.
+
+### Exercises
+The exercises associated with each topic allow students to gain experience with the skills in this course. The exercises are not assessed summatively (with a grade), but they may provide formative feedback to the students. After completing the exercises, students are equipped to start working on the course deliverables (graded assignments). The exercises are aimed for students to practice learning objectives c and d: ‘They can efficiently organize a reproducible programming and development process’ and ‘They can produce repositories up to the standards of international programming and coding conventions and initiatives’.
+
+### Deliverables
+The deliverables are graded assignments, collected in a portfolio. Students start working on the deliverables during the course, and can request feedback from the instructors and their peers. An overview of the deliverables is given in the section Assessment and feedback.
+The deliverables are handed in after the course ends, collected in a zip folder with their complete portfolio. During the process of working on deliverables, students will progress towards learning objectives b and e: ‘They can determine the most effective markup strategies to address a typesetting problem’ and ‘They can produce publications up to the typesetting standards of international peer- reviewed journals’. The requirements for the portfolio can be found under Assessment and feedback.
+
+### Procedure
+The lectures in this course have mandatory attendance. After each lecture, students make exercises, which are handed in week-by-week. Throughout the course, starting after some of the topics, students work on deliverables, which are handed in in a portfolio after the course ends. Students combine all exercises and deliverables into a portfolio, which is handed in after course ends. Students are allowed to make use of generative AI in this course, see Appendix II: GenAI rules.
```

Comment:
Instead of pointing to Appendix II, it would be better to point to the relevant section of this website